README.md

```diff
@@ -1,9 +1,9 @@
-# <span id="N4Mintroduction">NFDI4Microbiota - Metadata Standards</span> <br><br>
+NFDI4Microbiota - Metadata Standards
 
 <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />
-<br><br> This work is licensed under a
-<a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative
-Commons Attribution 4.0 International License</a>.
+This work is licensed under a
```

Comment:
Thanks for starting to cleanup but you went a bit too far and removed the URL to the license. Below is a suggestion on how to do such a link using the Markdown syntax.

```suggestion
This work is licensed under a [Creative Commons Attribution 4.0 International License](http://creativecommons.org/licenses/by/4.0/).
```